casual/casual-test/src/test/groovy/se/laz/casual/test/CasualEmbeddedServerTest.groovy

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2024, The casual project. All rights reserved.
+ *
+ * This software is licensed under the MIT license, https://opensource.org/licenses/MIT
+ */
+
+package se.laz.casual.test
+
+
+import spock.lang.Specification
+
+class CasualEmbeddedServerTest extends Specification
+{
+    CasualEmbeddedServer instance
+
+    def setup()
+    {
+        instance = CasualEmbeddedServer.newBuilder().build()
+    }
+
+    def cleanup()
+    {
+        if ( instance != null && instance.isRunning() )
+        {
+            instance.shutdown()
+        }
+    }
+
+    def "Create then get"()
```

Comment:
erm, yeah it does, but the `getters` are `is` because they are `boolean`s, the last is a `get`. I can update the test "create then check state" perhaps?